components/post_view/post_body/post_body.jsx

```diff
@@ -108,14 +118,6 @@ export default class PostBody extends React.PureComponent {
         this.sendingAction.cancel();
     }
 
-    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
-        const post = nextProps.post;
-        if (post && post.id !== post.pending_post_id) {
-            this.sendingAction.cancel();
```

Comment:
Do we still have to cancel `sendingAction` after `this.state.sending` is set to false?